build.xml

```diff
@@ -469,6 +469,10 @@
                     <arg value="-r"/>
                     <arg value="copy('${path.upgrade-notes}/_template.md', '${path.upgrade-notes}/backend_' . date('Ymd_His') . '.md');"/>
                 </exec>
+                <exec executable="php" escape="false" passthru="true">
+                    <arg value="-r"/>
+                    <arg value="echo 'File generated: ${path.upgrade-notes}/backend_' . date('Ymd_His') . '.md';"/>
```

Comment:
Love the idea, however, is not there a danger of changing the time between generating the file and outputting the name? I would rather save the file name in a property and then output it :thinking: This is what I got from chatbot (not tested though):
```
<!-- Set the timestamp in a property -->
                <exec executable="php" outputproperty="currentTimestamp">
                    <arg value="-r"/>
                    <arg value="echo date('Ymd_His');"/>
                </exec>
                
                <!-- Set the filename using the timestamp property -->
                <property name="upgradeNotesFilename" value="${path.upgrade-notes}/backend_${currentTimestamp}.md"/>
                
                <!-- Generate the file -->
                <exec executable="php">
                    <arg value="-r"/>
                    <arg value="copy('${path.upgrade-notes}/_template.md', '${upgradeNotesFilename}');"/>
                </exec>
                
                <!-- Output the filename -->
                <exec executable="php" escape="false" passthru="true">
                    <arg value="-r"/>
                    <arg value="echo 'File generated: ${upgradeNotesFilename}';"/>
                </exec>
```

Reply:
I have reworked it